cardano-cli/ChangeLog.md

```diff
@@ -1,5 +1,95 @@
 # Changelog for cardano-cli
 
+## 1.34.0 -- February 2022
+
+- Fix some spelling errors in the CLI help text.  (#3499)
+- Add a prettier rendering of update proposals. (#3208)
+- Add support for CBOR-encoded blobs in the `transaction build` and `transaction
+  build-raw` commands. (#3483)
+- Implement a `leadership-schedule` command. This can calculate a stake pool's
+  leadership schedule for the current and following epoch. It requires access to
+  the VRF signing key for that stake pool.
+
+  ```
+  > cardano-cli query leadership-schedule \
+     --testnet-magic 42 \
+     --genesis example/shelley/genesis.json \
+     --stake-pool-id  pool12t0y7agkqct89pf00eeytkvfjlquv76tjy27duannan9w63ckxv \
+     --vrf-signing-key-file example/node-pool1/shelley/vrf.skey
+     --current
+
+     SlotNo                          UTC Time
+     --------------------------------------------------------
+     4073                   2021-12-29 17:26:54.998001755 UTC
+     4126                   2021-12-29 17:27:00.298001755 UTC
+     4206                   2021-12-29 17:27:08.298001755 UTC
+     4256                   2021-12-29 17:27:13.298001755 UTC
+     4309                   2021-12-29 17:27:18.598001755 UTC
+     4376                   2021-12-29 17:27:25.298001755 UTC
+     4423                   2021-12-29 17:27:29.998001755 UTC
+     4433                   2021-12-29 17:27:30.998001755 UTC
+  ``` (#3464, #3494)
+- The CLI now supports outputting transaction bodies in ledger-compliant CDDL in
+  the `transaction build` and `transaction build-raw` commands. This is
+  specified by using the `--cddl-format` flag. (#3505)
+- Implement a `kes-period-info` command in the CLI. This checks that your
+  operational certificate is correct. It checks:
+  - The counters match what is in the node's protocol state
+  - The KES period in the operational certificate is correct (based on the
+    current slot).
+  ```
+  > cardano-cli query kes-period-info --testnet-magic 42  \
+    --op-cert-file example/node-pool1/shelley/node.cert
+  ✓ The counters match what is in the node's protocol state
+  ✓ The counters in the operational certificate and operational certificate issue counter file are the same
+  ✓ Operational certificate's kes period is within the correct KES period interval
+  {
+      "qKesNodeStateOperationalCertificateNumber": 6,
+      "qKesCurrentKesPeriod": 404,
+      "qKesOnDiskOperationalCertificateNumber": 6,
+      "qKesRemainingSlotsInKesPeriod": 3760228,
+      "qKesMaxKESEvolutions": 62,
+      "qKesKesKeyExpiry": "2022-03-20T21:44:51Z",
+      "qKesEndKesInterval": 434,
+      "qKesStartKesInterval": 372,
+      "qKesSlotsPerKesPeriod": 129600
+  }
```

Comment:
This needs to be updated to:
```
✓ The operational certificate counter agrees with the node protocol state counter
✓ Operational certificate's kes period is within the correct KES period interval
{
    "qKesNodeStateOperationalCertificateNumber": 6,
    "qKesCurrentKesPeriod": 404,
    "qKesOnDiskOperationalCertificateNumber": 6,
    "qKesRemainingSlotsInKesPeriod": 3760228,
    "qKesMaxKESEvolutions": 62,
    "qKesKesKeyExpiry": "2022-03-20T21:44:51Z",
    "qKesEndKesInterval": 434,
    "qKesStartKesInterval": 372,
    "qKesSlotsPerKesPeriod": 129600
}
```

Reply:
Darn, not sure how I missed that. Thanks!